src/utils.js

```diff
@@ -197,10 +197,7 @@ export function scrollIntoView(
     scrollTo(
       menuEl,
       Math.min(
-        focusedEl.offsetTop +
```

Comment:
Oh lord, they specifically went *out of their way* to do the current behavior?

Reply:
It looks like...